src/OpenTelemetry.AutoInstrumentation/Configurations/ResourceConfigurator.cs

```diff
@@ -26,6 +26,8 @@ public static void Configure(ResourceBuilder resourceBuilder)
 
```

Comment:
nit: not from your change but this local, `pluginManager` (line 25), should be defined near the if where the variable is used.